courses/machine_learning/deepdive2/supplemental/solutions/object_detection_with_automl.ipynb

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
<ul><li>Remove <code>from google.cloud import automl</code>  (It has already been imported earlier)</li><li><code>path</code>  -&gt; <code>DATASET_URI</code></li><li> Remove the 2 first comments</li><li>Get the <code>dataset_id</code>  directly from the <code>dataset</code>  object you constructed instead of asking to manually set it</li><li>Remove <code>client = automl.AutoMlClient()</code>  (you have already instantiated the client earlier)</li><li><code>us-central1</code>  -&gt; <code>ZONE</code> </li><li>Why do you have <code>input_uris = path.split(",")</code> ? It seems unnecessary, since your <code>path</code>  (should be renamed <code>DATASET_URI</code>)  does not contain any <code>,</code> . I'd suggest your remove it. </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>dataset_id = created_dataset.name.split("/")[-1] </p><p> </p><p>sends out error "created_dataset not defined"</p><p><a href="https://screenshot.googleplex.com/UJ9Jm0nQ7M5" target="_blank">https://screenshot.googleplex.com/UJ9Jm0nQ7M5</a></p><p> </p><p>leaving DATASET_ID = "IOD7192557567678087168" for now</p><p> </p><p>all other items are actioned</p>

Reply:
<p>disregard previous comment. I've got the code snippet working with:</p><p> </p><p><code>DATASET_ID = format(created_dataset.name.split("/")[-1])</code></p>